op-service/eth/account_proof_test.go

```diff
@@ -126,11 +126,12 @@ func TestAccountResult_MarshalTruncated(t *testing.T) {
 }
 
 func FuzzAccountResult_StorageProof(f *testing.F) {
+	f.Skip() // TODO https://github.com/ethereum-optimism/optimism/issues/15067
 	f.Fuzz(func(t *testing.T, key []byte, value []byte) {
 		result := makeResult(t)
 		result.StorageProof[0].Key = key
 		result.StorageProof[0].Value = hexutil.Big(*(new(big.Int).SetBytes(value)))
-		require.NotNil(t, result.Verify(goodRoot), "does not verify against bad proof")
+		require.NoError(t, result.Verify(goodRoot), "does not verify against bad proof")
```

Comment:
Perhaps we should be expecting an error here? It seems that this should fail, if we modify a known result and root with arbitrary data, we should not expect the result to verify?

Reply:
edit: I realise I got confused here. Will fix.

Reply:
I used git bisect to work out that this fuzz test broke with this PR https://github.com/ethereum-optimism/optimism/pull/7227. 

[b42767b](https://github.com/ethereum-optimism/optimism/pull/15068/commits/b42767bdf799890548536a6cd75a5a69cdad09dc) fixes the assertions in the fuzz test to match the change in behaviour introduced above.